activefires_pp/spatiotemporal_alarm_filtering.py

```diff
@@ -398,36 +433,44 @@ def create_single_point_alarms_from_collections(features):
     return fcollection
 
 
-def get_single_point_fires_as_collections(fires, threshold):
+def get_single_point_fires_as_collections(fires, long_fires_threshold):
     """Split larger fires into smaller parts and make a single fire-detection out of each and return as collection."""
-    features = split_large_fire_clusters(fires, threshold)
+    features = split_large_fire_clusters(fires, long_fires_threshold)
     return create_single_point_alarms_from_collections(features)
 
 
 def check_if_fire_should_trigger_alarm(gjson_data, past_alarms_dir, sos_alarms_file_pattern,
-                                       hour_thr=16, km_threshold=0.8):
+                                       time_space_thresholds):
     """Check if fire point should trigger an alarm.
 
-    The fire point is a GeoJSON object. A search back in time X hours (X=16) is
+    The fire point is a GeoJSON object. A search back in time X hours (e.g. X=16) is
     done, and a check for previous fire alarms on that location is done. Only
     if no previous fire alarm at the same position (determined by a distance
-    threshold) is found an alarm should be issued.
+    threshold) is found, an alarm should be issued.
     """
     utc = pytz.timezone('utc')
     end_time = datetime.fromisoformat(gjson_data["properties"]["observation_time"])
     end_time = end_time.astimezone(utc).replace(tzinfo=None)
 
+    hour_thr = time_space_thresholds.get('hour_threshold', 16)
+    km_threshold = time_space_thresholds.get('spatial_threshold_km', 0.8)
+
+    LOG.debug("Check if fire detection should trigger alarm. Thresholds = (%3.1f h, %3.1f km)",
+              hour_thr, km_threshold)
+
     start_time = end_time - timedelta(hours=hour_thr)
-    recent_files = get_recent_geojson_files(past_alarms_dir, sos_alarms_file_pattern, (start_time, end_time))
+    recent_files = get_recent_geojson_files(past_alarms_dir, sos_alarms_file_pattern,
```

Comment:
To make things clearer, I suggest changing the names of the variable and the function so that is it clear the files are already sorted by age. eg `files_in_observation_time_order` and `get_geojson_files_in_observation_time_order`.

Reply:
Thanks, good suggestion!